test/PendingOwnableUpgradeable.js

```diff
@@ -0,0 +1,93 @@
+const { config, ethers, network } = require("hardhat");
+const { expect } = require("chai");
+
+describe("MockPendingOwnableUpgradeable", function () {
```

Comment:
Same, add a failing test trying to initialize again. (that way you would spot if you'd have forgotten the onlyIntialize modfier)